pkg/sdn/plugin/sdn-cni-plugin/openshift-sdn.go

```diff
@@ -0,0 +1,463 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"syscall"
+
+	sdnplugin "github.com/openshift/origin/pkg/sdn/plugin"
+	"github.com/openshift/origin/pkg/sdn/plugin/api"
+	"github.com/openshift/origin/pkg/util/netutils"
+
+	oclient "github.com/openshift/origin/pkg/client"
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	osapi "github.com/openshift/origin/pkg/sdn/api"
+
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	knetwork "k8s.io/kubernetes/pkg/kubelet/network"
+	kbandwidth "k8s.io/kubernetes/pkg/util/bandwidth"
+
+	"github.com/containernetworking/cni/pkg/ip"
+	"github.com/containernetworking/cni/pkg/ipam"
+	"github.com/containernetworking/cni/pkg/ns"
+	"github.com/containernetworking/cni/pkg/skel"
+
+	"github.com/vishvananda/netlink"
+
+	// Must be imported to register API types like ClusterNetwork
+	_ "github.com/openshift/origin/pkg/api/install"
+)
+
+const (
+	sdnScript   = "openshift-sdn-ovs"
+	setUpCmd    = "setup"
+	tearDownCmd = "teardown"
+	statusCmd   = "status"
+	updateCmd   = "update"
+
+	AssignMacVlanAnnotation string = "pod.network.openshift.io/assign-macvlan"
```

Comment:
@openshift/cucushift-committers These are used between openshift-sdn and the shellscript that does some of the pod networking setup.  It is in no way public, simply constants that are agreed between pkg/sdn/plugin/sdn-cni-plugin/openshift-sdn.go and pkg/sdn/plugin/sdn-cni-plugin/openshift-sdn-ovs.